kratos/utilities/variable_utils.h

```diff
@@ -133,13 +133,12 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
                                                         << "\n\t- Number of origin nodes: " << n_orig_nodes
                                                         << "\n\t- Number of destination nodes: " << n_dest_nodes << std::endl;
 
-        #pragma omp parallel for
-        for(int i_node = 0; i_node < n_orig_nodes; ++i_node){
-            auto it_dest_node = rDestinationModelPart.NodesBegin() + i_node;
-            const auto &it_orig_node = rOriginModelPart.NodesBegin() + i_node;
+        IndexPartition<std::size_t>(n_orig_nodes).for_each([&](std::size_t index){
+            auto it_dest_node = rDestinationModelPart.NodesBegin() + index;
+            const auto &it_orig_node = rOriginModelPart.NodesBegin() + index;
```

Comment:
```suggestion
            const auto it_orig_node = rOriginModelPart.NodesBegin() + index;
```
In the other hand iterators don't require reference